packages/ses/test/hermes-smoke.js

```diff
@@ -0,0 +1,44 @@
+// TODO: equivalent of:
+// import '../dist/ses.cjs';
+// import '../src/assert-shim.js';
+// Since Hermes has no native support for I/O
+
+// Test lockdown
+
+lockdown();
+
+// Test Compartment
+
+const c = new Compartment();
+
+c.evaluate('1+1');
+c.evaluate("const c2 = new Compartment(); c2.evaluate('1+2')");
```

Comment:
remove

Reply:
done in https://github.com/leotm/endo/pull/1/commits/0def87db44b43ea7fddb16fa4489a4176502b3f8